CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 # VERSION HISTORY
 
-- 1.4.15 (in progress)
+- 1.5 (in progress)
+  - Fix an assertion error in chrF (#121)
```

Comment:
Make sure to change this to the data (2021-01-12 is fine even if we don't end up approving today)

Reply:
Done, changed to 15 January.